dev/archery/requirements-lint.txt

```diff
@@ -1,2 +1,3 @@
+autopep8
 flake8
```

Comment:
We can get rid of flake8?

Reply:
No, because we still use it (it catches non-style issues).